base_report_to_printer_mail/readme/CREDITS.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
If the file's empty, just remove it :)

Reply:
ok - kicked the file - was not sure about this